src/Bugsnag/Middleware.cs

```diff
@@ -78,14 +78,17 @@ public static class InternalMiddleware
         {
           foreach (var stackTraceLine in exception.StackTrace)
           {
-            foreach (var @namespace in report.Configuration.ProjectNamespaces)
+            if (!Polyfills.String.IsNullOrWhiteSpace(stackTraceLine.MethodName))
```

Comment:
I think it would be worth adding another unit test case to cover the null case, given that we have a unit test framework in place for the class.